NEWS.md

```diff
@@ -40,6 +40,10 @@
         is changed to `$versions$r_package` (#791).
     -   `$rust_polars`, which indicates the version of the dependent Rust Polars,
         is changed to `$versions$rust_crate` (#791).
+-   New behavior when creating a `DataFrame` with a single list-variable. 
```

Comment:
```suggestion
-   New behavior when creating a `DataFrame` with a single list-variable.
```

Reply:
Can we avoid this type of changes? This is something that `task format-r` doesn't fix

Reply:
An auto-formatter for Markdown must be installed.
VSCode can easily handle this with extensions, but I am not familiar with the tools available for other editors. Perhaps Pretitter is well known?

Reply:
I don't know how that works with RStudio